source/includes/_batch-operations.md

```diff
@@ -104,6 +104,25 @@ and `always_update` to bypass the validation procedure.
   GRESB Asset Portal.
 </aside>
 
+Please be aware that access to this endpoint is limited in order to prevent abuse
+or overuse caused by a buggy or malicious client. It happens in two ways:
+
+  - Not more than 10 requests are allowed during 10 minutes.
+  - Each of the fields in the request is limited to 2000 assets at once.
+
+These values are experimental and therefore are subject to change.
+However, you are unlikely to reach them during normal operation.
+
+The information about the throttling is also provided in the following headers:
+
+  - `X-RateLimit-Limit` contains the maximum number of allowed requests during a time period.
+  - `X-RateLimit-Reset` expected counter reset time in UTC epoch seconds.
+
+If a response is over the limit, it's rejected with 429 status and an additional header is present:
```

Comment:
Is the header only present when a response is over the limit or is it always present?

Reply:
The former - only when over the limit. 
It's a bit tough to calculate it for the exact value. It's possible, but would require some additional points invested, which means that so far we know only when they are exhausted.

Reply:
Well it was part of the spec to communicate the current usage / remaining requests. I also asked you in the [pull request](https://github.com/GRESB/gresb-reports/pull/7870#discussion_r295733163) if you tried a gem that seems to do it, but you didn't answer? Was it broken?

Reply:
I have answered there. It might not be broken now, but will break eventually because it's not maintained.